### ISSUE TITLE
continuous-control doesn't work for MountainCarContinuous-v0

### ISSUE THREAD

**Issue by Belerafon** [JamesChuanggg/pytorch-continuous-REINFORCE#3]
Hi. First af all - thanks for good clear code!
My problem.  I am trying to run this continuous-control algorithm "as is" for simplest gym's enviroments such as MountainCarContinuous-v0, Pendulum-v0 and nothing work. I tryed to change lr, number of episodes, quantity of neural network layers, and so on - nothing. There are random fluctuations on the end of learning near low-reward position. Can you provide good parametres for this enviroments or give recommendations what to do?
